workflows/se_utils/__init__.py

```
@@ -153,3 +152,32 @@ def get_branch_handle_from_url(lp_handle, url):
                             'lp://staging/')
         print('fetching branch: ' + name)
         return lp_handle.branches.getByUrl(url=name)
+
+
+def download_snap_build(lp_handle, buildUrl, destination):
+    """ Download a snap build from a url to a destination.
+    If the download fails, do not raise an exception, just return False.
+    :param buildUrl: url of the snap build to download
+    :param destination: path to save the downloaded file
+    :return: True if the download was successful, False otherwise
+    """
+    try:
+        snap_build = lp_handle.load(buildUrl)
+        urls = snap_build.getFileUrls()
+        if len(urls):
```

This should return an error if there are no URLs to download from.